src/index.js

```diff
@@ -31,6 +31,13 @@ import SettingsAssignedUsersRoute from './routes/settings-assigned-users-route';
 
 class EHoldings extends Component {
   static propTypes = {
+    history: PropTypes.shape({
```

Comment:
I think we should use `react-router-prop-types` here to handle the shape of `history`, `location` and `match` props